tests/AutoReview/FixerTest.php

```diff
@@ -83,14 +108,55 @@ public function testFixerDefinitions(FixerInterface $fixer)
             );
             $this->assertFalse(
                 $duplicatedCodeSample,
-                sprintf('[%s] Code sample #%d duplicates #%d.', $fixer->getName(), $sampleCounter, $duplicatedCodeSample)
+                sprintf('[%s] Code sample #%d duplicates #%d.', $fixerName, $sampleCounter, $duplicatedCodeSample)
             );
         }
 
+        if ($fixerIsConfigurable) {
+            if (isset($configSamplesProvided['default'])) {
+                reset($configSamplesProvided);
+                $this->assertSame('default', key($configSamplesProvided), 'First code sample must be for the default configuration.');
+            } else {
+                if (!isset($this->allowedRequiredOptions[$fixerName])) {
+                    $this->fail(sprintf('[%s] Has no sample for default configuration.', $fixerName));
```

Comment:
i wounder could we do that `assertSame + fail` with one assertion... i like to have assertions in tests, not `fail` directly, imo later shall be only inside assertion definition ;)